mobile/lib/http/test_list.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+class TestList {
+  final List testList;
+
+  const TestList({
+    required this.testList,
+  });
+
+  factory TestList.fromJson(Map<String, dynamic> json) {
+    return TestList(
+      testList: json['data'],
+    );
+  }
+}
+
+Future<TestList> fetchTestList() async {
+  final response = await http.get(Uri.parse('http://127.0.0.1:3000/api/v1/tests'));
+  if (response.statusCode == 200) {
+    return TestList.fromJson(jsonDecode(response.body));
```

Comment:
**THIS IS JIM**

Yo, this is f'in awesome. I don't know Dart at all and was quickly browsing this over with a friend next to me lol it's Sarah next to me. 

But, can `fromJson(jsonDecode:)` throw an exception? In theory, I would imagine a scenario where this decoder can't get this to a `TestList` object. But perhaps it just returns back something empty?

Goodbye.

@fpcyan

Reply:
Jim also says people use Fastlane for deploying, and here's a reference.  He recommends starting with it sooner than later so it can be incorporated in the project without disruption 
https://circleci.com/blog/deploy-flutter-android/?utm_source=google&utm_medium=sem&utm_campaign=sem-google-dg--uscan-en-dsa-tROAS-auth-nb&utm_term=g_-_c__dsa_&utm_content=&gclid=CjwKCAjwsvujBhAXEiwA_UXnADNRd_P57bIYVtBLlEB9pMwaHU_T4F7rkvOJZ5Xlgh4AekqmkgsiahoCLI0QAvD_BwE